index.js

```diff
@@ -1,5 +1,44 @@
 'use strict'
 
+function code(state, startLine, endLine/*, silent*/) {
+  var nextLine, last, token, match
+  if (state.sCount[startLine] - state.blkIndent < 4) { return false }
+  last = nextLine = startLine + 1
+  while (nextLine < endLine) {
+    if (state.sCount[nextLine] - state.blkIndent >= 4) {
+      nextLine++
+      last = nextLine
+      continue
+    }
+    break
+  }
+  state.line = last
+  token         = state.push('indented_quote', 'blockquote', 0)
+  token.content = state.getLines(startLine, last, 4 + state.blkIndent, true)
```

Comment:
We can't use `token.content` because it doesn't properly allow for inline markdown; instead we need to have something like the following token types:
- indented_quote_open
- inline
- indented_quote_close

—Check out however the paragraph rule does it for an example.

Reply:
I've added a test for proper inline parsing to the master branch, so that should clarify things.

Reply:
Fixed. Also I have changed spaces (charCode === 32)  in the definitions.txt' result section to nbsp's (charCode === 160) because otherwise the test could not be passed.